contracts/CreateAccount.sol

```diff
@@ -139,15 +139,15 @@ contract CreateAccount is FraudProofHelpers {
         bytes32 _accountsRoot,
         Types.CreateAccount memory _tx,
         Types.PDAMerkleProof memory _to_pda_proof,
-        Types.AccountProofs memory accountProofs
+        Types.AccountMerkleProof memory to_account_proof
```

Comment:
@vaibhavchellani No need passing `from` proofs here, so I removed the `from` proof and keep the `to` proof. Does this affect the node?

Reply:
Nah, should be okay. No problem.